fix: various broken links (#1465)
foundry-rs/book: src/forge/differential-ffi-testing.md

#### Standardized Testing against reference data

> FFI is also useful for injecting reproducible, standardized data into the testing environment. In the Murky library, this is used as a benchmark for gas snapshotting (see [forge snapshot](./gas-function-snapshots.md)).

Reviewer: ```suggestion
FFI is also useful for injecting reproducible, standardized data into the testing environment. In the Murky library, it is used as follows:
```
